src/app/forms/templates/general/defect.template.ts

```diff
@@ -136,9 +136,9 @@ export const DefectsTpl: FormNode = {
                     {
                       name: 'prs',
                       label: 'PRS',
+                      value: null,
                       type: FormNodeTypes.CONTROL,
                       editType: FormNodeEditTypes.RADIO,
-                      value: null,
```

Comment:
These values will need to be initialised as `null` for add defect to work as intended

Reply:
Oh, I did not remove it. Just that in my branch `value: null,` was a few lines above compared to yours.

Reply:
🤦